src/Control/Exception/Safe.hs

```diff
@@ -183,7 +226,7 @@ instance Exception StringException
 -- <https://github.com/fpco/safe-exceptions#quickstart>
 --
 -- @since 0.1.0.0
-throwTo :: (Exception e, MonadIO m) => ThreadId -> e -> m ()
+throwTo :: HAS_CALL_STACK => (Exception e, MonadIO m) => ThreadId -> e -> m ()
 throwTo tid = liftIO . E.throwTo tid . toAsyncException
```

Comment:
This one is probably unnecssary, since `E.throwTo` doesn't do anything with `CallStack` information. It also may not, depending on how the GHC proposal goes.